webhook/logging/logging.go

```diff
@@ -60,6 +60,6 @@ var WebhookLogger = func(errorType string, message interface{}) error {
 	log.SetOutput(multi)
 
 	logEntry := fmt.Sprintf("%s - %s - %s\n", errorType, currentDateTime(), messageString)
```

Comment:
@koladev32, have you tried it? Apparently, it will produce the following output:
```
 ERROR_TYPE - YYYY-MM-DD HH:MM:SS
 - ERROR_STRING

 ERROR_TYPE - YYYY-MM-DD HH:MM:SS
 - ERROR_STRING
```
Why ? The `"\n"`

```suggestion
          logEntry := fmt.Sprintf("%s - %s - %s", errorType, currentDateTime(), messageString)
```
just delete the `\n`

Reply:
Well, complicated. The `\n` is needed to go into a new line to continue logging